src/System.Memory/ref/System.Memory.cs

```diff
@@ -424,15 +490,11 @@ namespace System.Runtime.InteropServices
     public static partial class MemoryMarshal
     {
         public static System.Memory<T> AsMemory<T>(System.ReadOnlyMemory<T> readOnlyMemory) { throw null; }
+        public static System.ReadOnlySpan<TTo> Cast<TFrom, TTo>(System.ReadOnlySpan<TFrom> source) where TFrom : struct where TTo : struct { throw null; }
```

Comment:
Why are these casts showing in your PR?

Reply:
I don't know why but ref generation on my machine reordered things.

Reply:
I think I'll just edit things manually

Reply:
Fixed.